extensions/empty-screenshots/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { exec } from "child_process";
+import { readdir, stat, rm } from "fs/promises";
+import { join } from "path";
+import { promisify } from "util";
+import { showToast, Toast, closeMainWindow, showHUD, getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  daysToKeep: string;
+}
+
+const asyncExec = promisify(exec);
+
+export default async function main() {
+  const { daysToKeep } = getPreferenceValues<Preferences>();
+  const { stdout, stderr } = await asyncExec(
+    "defaults read com.apple.screencapture | plutil  -convert json -r -o - -- -"
+  );
+
+  if (stderr) {
+    showToast(Toast.Style.Failure, "Something went wrong", stderr);
+  }
+
+  const numberOfDays = parseInt(daysToKeep, 10);
+  const { location } = JSON.parse(stdout);
```

Comment:
in my case, `stdout` is:
```
{
  "last-analytics-stamp" : "682863474.458861",
  "last-selection" : {
    "Height" : "315",
    "Width" : "364",
    "X" : "453",
    "Y" : "0"
  },
  "last-selection-display" : "0",
  "style" : "selection",
  "video" : "1"
}
```
so `location` ends up being undefined, crashing the extension 😅 

<img width="839" alt="image" src="https://user-images.githubusercontent.com/1155589/186892354-538cdafe-dfff-46cb-8c7a-ecd5dad8f371.png">

What about making this a preference instead?

Reply:
Just out of curiosity, how do you even set it? I tried to look in macOS Ventura for it and I'm not sure where to find it (Also getting the same error as @sxn)

Reply:
Not sure about Ventura, since I'm still on Monterey, but [here's an article](https://www.macworld.com/article/673251/how-to-change-where-screenshots-are-saved-on-a-mac.html#how-to-change-screenshot-location-on-a-mac) on how to do it on other OS versions.

Reply:
We're not all in Ventura actually, only a few 🙂 I think the best approach here would be to add a preference value where you can set the directory path (maybe with a tilde(~) support)

Reply:
Done (and done)! 😄 Let me know if there are any other concerns 🙏